Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.12.3-slim
+LABEL maintainer="94nj111@gmail.com"
+
+ENV PYTHONDONTWRITEBYTECODE=True
+ENV PYTHONUNBUFFERED=True
+
+WORKDIR /app
+
+ARG UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/app" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    appuser
+
+RUN pip install --no-cache-dir requests
```

Comment:
Consider installing all required Python packages from a `requirements.txt` file for better dependency management. This approach makes it easier to manage dependencies and update them as needed.